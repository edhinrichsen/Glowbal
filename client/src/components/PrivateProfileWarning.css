.private-profile-warning {
  display: flex;
  justify-content: center;
  align-items: center;
  height: 80vh;
}
