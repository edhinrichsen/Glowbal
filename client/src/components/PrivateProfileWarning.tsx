import React from 'react';
import { Button, Result } from 'antd';
import { useHistory } from 'react-router-dom';

import './PrivateProfileWarning.css';

function PrivateProfileWarning() {
  let history = useHistory();
  return (
    <div className="private-profile-warning">
      <Result
        status="warning"
        title="This profile is set to private"
        extra={
          <Button type="primary" onClick={() => history.goBack()}>
            Go Back
          </Button>
        }
      />
    </div>
  );
}

export default PrivateProfileWarning;
