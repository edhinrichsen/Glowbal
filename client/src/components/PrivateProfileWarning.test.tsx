import React from 'react';
import { shallow } from 'enzyme';
import PrivateProfileWarning from './PrivateProfileWarning';

describe('Testing PrivateProfileWarning', () => {
  it('renders without crashing', () => {
    const component = shallow(<PrivateProfileWarning />);
    expect(component).toMatchSnapshot();
    component.unmount();
  });
});
