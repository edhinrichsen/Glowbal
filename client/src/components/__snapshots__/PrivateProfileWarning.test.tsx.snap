// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing PrivateProfileWarning renders without crashing 1`] = `
<div
  className="private-profile-warning"
>
  <Result
    extra={
      <ForwardRef(InternalButton)
        block={false}
        ghost={false}
        htmlType="button"
        loading={false}
        onClick={[Function]}
        type="primary"
      >
        Go Back
      </ForwardRef(InternalButton)>
    }
    status="warning"
    title="This profile is set to private"
  />
</div>
`;
